src/components/joinus/JoinUsPage.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import MarkDownRenderer from '../commons/renderer/MarkDownRenderer';
+import joinUsFile from './JoinUs.md';
+
+const JoinUsPage = () => {
+    const [markDownText, setMarkDownText] = useState('');
```

Comment:
Why you put in a state? The markdown is static, so we don't need a state here.

Reply:
By using a state, the page will render while the MarkDown text is being obtained by `fetch`, and also I can pass its value as props for `MarkDownRenderer` component.